test/src/test/java/jenkins/model/NodesTest.java

```diff
@@ -287,6 +287,7 @@ public static class CheckSetNodes extends NodeListener {
 
         @Override
         protected void onCreated(@NonNull Node node) {
+            node.getRootDir();
```

Comment:
I don't think this is explicitly required for this change? I'm guessing this is some testing code?

Reply:
Without changes in `src/main/java` this throws an exception.

Reply:
Ah got it. Makes sense to me